Makefile

```diff
@@ -781,10 +797,20 @@ COMMON_SRC = \
             io/vtx_string.c \
             io/vtx_smartaudio.c \
             io/vtx_tramp.c \
-            io/vtx.c \
+            io/vtx.c
+            
+COMMON_DEVICE_SRC = \
             $(CMSIS_SRC) \
             $(DEVICE_STDPERIPH_SRC)
 
+ifeq ($(OSD_SLAVE),yes)
+TARGET_FLAGS := -DUSE_OSD_SLAVE $(TARGET_FLAGS)
+COMMON_SRC := $(COMMON_SRC) $(OSD_SLAVE_SRC) $(COMMON_DEVICE_SRC)
+else
+TARGET_FLAGS := -DUSE_FC $(TARGET_FLAGS)
```

Comment:
I don't really like `USE_FC` since:
1. it is the assumed default
2. It is complementary to `USE_OSD_SLAVE`

So I'd rather see `#ifndef USE_OSD_SLAVE` than `#ifdef USE_FC` in the code.

In fact, thinking about it, if `USE_FC` is used, then, in principle, the whole codebase should be under an `USE_FC`. So I really don't like it at all.